code/modules/spells/targeted/necromancer_spells.dm

```diff
@@ -103,11 +103,14 @@
 			to_chat(user, "This spell can't affect non-organics.")
 			return 0
 
+		//Save the mind so we can give them first dibs
+		var/datum/mind/M = target.mind
 		var/mob/living/carbon/human/skeleton/F = new(get_turf(target))
 		target.visible_message("<span class='cult'>\The [target] explodes in a shower of gore, a skeleton emerges from the remains!</span>")
 		target.gib()
 		var/datum/ghosttrap/ghost = get_ghost_trap("skeleton minion")
-		ghost.request_player(F,"A wizard is requesting a skeleton minion.", 60 SECONDS)
+		if(!M || !ghost.transfer_personality(M.current,F))
```

Comment:
```dm
/mob/living/can_be_possessed_by(var/mob/abstract/observer/possessor)
	if(!..())
		return 0
	if(!possession_candidate)
		to_chat(possessor, "<span class='warning'>That animal cannot be possessed.</span>")
		return 0
	if(jobban_isbanned(possessor, "Animal"))
		to_chat(possessor, "<span class='warning'>You are banned from animal roles.</span>")
		return 0
	if(!possessor.MayRespawn(1,ANIMAL))
		return 0
	return 1
```

The MayRespawn() check might fail immediately. This making this not function. testing required.